.gitmodules

```diff
@@ -14,3 +14,7 @@
 	path = library/securimage
 	url = https://github.com/mantisbt/securimage.git
 	branch = mantis
+[submodule "library/parsedown"]
+	path = library/parsedown
+	url = https://github.com/erusev/parsedown.git
+	branch = mantisbt
```

Comment:
It's technically possible, but using our own fork allows defining of a custom branch where we can e.g. remove unwanted files such as test scripts.